pkg/action/uninstall.go

```diff
@@ -196,7 +196,7 @@ func joinErrors(errs []error) string {
 // deleteRelease deletes the release and returns list of delete resources and manifests that were kept in the deletion process
 func (u *Uninstall) deleteRelease(rel *release.Release) (kube.ResourceList, string, []error) {
 	var errs []error
-	caps, err := u.cfg.getCapabilities()
+	caps, err := u.cfg.getCapabilities(true)
```

Comment:
I have opened a separate PR to optimize the uninstall action (it doesn't need capabilities at all):
https://github.com/helm/helm/pull/13113